SF_WHOLE is to expect sfwrite() and sfputs write the data to the device
immediately.  So Count == 100 or no buffering is expected.

Index: src/lib/sfio/Sfio_t/twhole.c
--- src/lib/sfio/Sfio_t/twhole.c.orig
+++ src/lib/sfio/Sfio_t/twhole.c
@@ -42,7 +42,7 @@ MAIN()
 		if(sfwrite(f,s,52) != 52)
 			terror("sfwrite failed\n");
 	sfclose(f);
-	if(Count != 10)
+	if(Count != 100)
 		terror("Wrong number of writes1\n");
 
 	Count = 0;
@@ -58,7 +58,7 @@ MAIN()
 		if(sfputr(f,s,'\n') != 53)
 			terror("sfputr failed\n");
 	sfclose(f);
-	if(Count != 10)
+	if(Count != 100)
 		terror("Wrong number of writes2\n");
 
 	TSTEXIT(0);
